bootstrap.py

```diff
@@ -61,190 +69,109 @@
                     help="Comma-separated module name hierarchies whose log "
```

Comment:
`--no-apport` is long-obselete and was removed elsewhere, so it can be removed from the CLI

Reply:
What about `--show-console`: "(Deprecated) Does nothing, now the default behavior is to show the console"

Reply:
May as well remove that too, I guess, since it has been deprecated for at least a few years now IIRC